daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -35,6 +37,33 @@ class OrderingSpec
     with ScalaCheckDrivenPropertyChecks
     with ScalaCheckPropertyChecks {
 
+  private[lf] def toAstType(typ: iface.Type): Ast.Type = typ match {
```

Comment:
I assume we don’t have instances to generate `Ast.Type` directly instead of first generating iface types and then converting back?

Reply:
We don't, and if `toAstType` is half an isomorphism we're better off not having one.